router/protocol_fee_swap.gno

```diff
@@ -11,14 +11,17 @@ import (
 	u256 "gno.land/p/gnoswap/uint256"
 )
 
+const (
+	defaultSwapFee = uint64(15) // 0.15%
```

Comment:
afaik, it should be value of bps
@onlyhyde need double check

Reply:
yes i agree with you
we use bps unit for percent fee

Reply:
https://github.com/gnoswap-labs/gnoswap/pull/431/commits/cb187e76b71b14df0a6703debb988a92ac1f437d